src/Show.jl

```diff
@@ -81,7 +81,7 @@ For example this excludes characters created with [`tensor!`](@ref) or [`lincomb
 julia> g=genchartab(\"GL2\");
 
 julia> nrirrchars(g)
-q^2 - 1
+q^2 - 1//1
```

Comment:
Could we omit the `//1` in this case?

Reply:
Yes, we could and I think we should eventually. I need to fix the whole printing of the `CycloFrac` type anyways...